shared/messageStructure.js

```diff
@@ -0,0 +1,35 @@
+function messageNewUser(message, nickname) {
+  let newuser = {
+    message,
+    nickname
+  }
+  return newUser
+}
+
+function messageStartGame(message, nickname) {
+  let startGame = {
+    message,
+    nicknamen
+  }
+  return starGame
+}
+
+obj = {
+  messageNewUser,
+  messageStartGame
+}
+
+module.exports = obj
+
+/** README
+ * Para acessar um dos objetos e seus atributos,
+ * na classe server tu usa a variavel MessageStructure(Linha 4).
+ *
+ * EX: MessageStructure.starGame.replyMessage
+ *
+ * Inicialmente os atributos do objeto (EX:startGame) vão estar vazios.
+ *
+ * Depois de adicionar os valores deles, o objeto (EX:startGame) pode ser
+ * passado como resposta, depois de transformalo em string.
+ *
+ */
```

Comment:
Deveria estar no início do arquivo para leitura melhor. Se esse arquivo tivesse 4124532248749523161487765416135 linhas de código iria ficar difícil de ler a documentação